libs/gi/db/src/Interfaces/BuildTc.ts

```diff
@@ -17,6 +18,7 @@ export type BuildTcArtifactSlot = {
 export type BuildTc = {
   name: string
   description: string
+  character?: Omit<ICharacter, 'key'>
```

Comment:
More minor, but also no golden numbers here to show these are overridden values
![image](https://github.com/frzyc/genshin-optimizer/assets/36019388/3245dbd7-76a5-4658-bb33-8269336a7d2a)

Reply:
![Screenshot 2024-05-22 013432](https://github.com/frzyc/genshin-optimizer/assets/1754901/f420ac93-af19-482e-a511-03b5250e4561)